src/components/protection/AddProtectionSingle.vue

```diff
@@ -136,69 +136,54 @@
       :disabled="disableActionButton"
     />
 
-    <modal-base
-      :title="`${$t('staking_protecting')}:`"
-      v-model="modal"
-      @input="setDefault"
+    <modal-tx-action
+      title="Confirm Stake & Protect"
```

Comment:
Should we move this to i8ln ?

Reply:
ah yeah damn forgot about that. Will create an issue and fix that as a follow up. 

Lets get this out the door if it passes review.